### ISSUE TITLE
cant  find setting button on examples page 

### ISSUE THREAD

**Issue by madtitan02** [processing/p5.js-web-editor#2655]
### Increasing Access

Currently , Switching themes feature is available inside the setting button which is available only on the homepage.But when we switch to the examples section there is no setting button available o we cannot switch theme or change font size from other pages except the home page.We can provide the setting button on everypage as it will be very compatible for the user experience.As screenshot is attached below there is no setting button available we can simple add it to the right top where it is present on the homepage
![Screenshot from 2023-11-30 17-02-12](https://github.com/processing/p5.js-web-editor/assets/96717231/83451a85-c8e9-4dd2-95f3-178bc97c5daf)

### Feature request details

By simply adding the setting button on the right top corner will do the work we can simply embedd the feature there which is prexisting we just need to add it on the examples page

**Comment by madtitan02** [comment]
if the community agrees please do assign me i will work on it

**Comment by lindapaiste** [comment]
I’m on board with the principal.  I’m adding the “needs design” tag to this issue because we need to determine what exactly it should look like.  I don’t think that the huge gear button that we have on the home page would work here.  I’m thinking either the word “Settings” on the right side of the Nav bar or a smaller gear icon in the Nav bar.

Most of the settlings only apply to the IDE.  I think theme is the only one that is “global”? There might be a bit of a usability question regarding what to show in the popup and if it’s confusing to show IDE settings when on other pages.  It’s definitely easier to implement if we show the entire existing Preferences modal.  And that seems fine to me.

**Comment by gauravsingh94** [comment]
@lindapaiste 
In my opinion, incorporating the entire settings menu into the navigation bar may not be the optimal choice, as it includes numerous settings that are irrelevant to other pages.

**Proposed Approach 1:** We could relocate the theme enabler from the settings menu to the navigation bar.

**Proposed Approach 2:** Separate the IDE settings from the settings and then integrate the settings into the navigation bar.

**Comment by nick2432** [comment]
Can I contribute to this issue? I'm new here, but I have experience contributing to many other organizations. Is there a Discord channel or any other channel where I can discuss and get assistance?

**Comment by lindapaiste** [comment]
> Can I contribute to this issue? I'm new here, but I have experience contributing to many other organizations. Is there a Discord channel or any other channel where I can discuss and get assistance?

There is "contribute-to-p5" section on the [p5.js discord](https://discord.gg/SHQ8dH25r9).

For this issue I think we need to finalize the design first before coding.

**Comment by adadgarg** [comment]
Hi @lindapaiste, Here is a  proposed design:
We can add more items along with the theme as per the requirement.

![2 (1)](https://github.com/processing/p5.js-web-editor/assets/104292766/c440ee90-becd-49a4-8c9f-cda995074ed4)

![group2](https://github.com/processing/p5.js-web-editor/assets/104292766/c9452044-08ee-4d21-bdb9-9783bb651b7b)

**Comment by Akshun-01** [comment]
Hello, for this issue I have some suggestions: 
- First if we are planning to add this feature as a general feature and not only restricted to homepage, then i think **adding it to the navbar** will do the trick.(as illustrated below)
It will make sure we don't have to add it manually to any other page we decide to add in the future as and would save code repetition and rendering time as well. 
![image](https://github.com/processing/p5.js-web-editor/assets/93793691/0e1742fa-bc8c-4d24-9d8a-16011c18e2f2)

- For the same we also have the option to add it to the left side of the navbar but we are infact changing that part from all the options available to "back to editor", so it might look weird. We have the following design ideas, if we consider **aligning it to the left in navbar.**(below illustrations)
![image](https://github.com/processing/p5.js-web-editor/assets/93793691/b438449b-c695-414d-951b-1a463d09a860)

- Now if we want to add it **only to the examples page**, then i think what aditya has mentioned is perfectly fine to go with. We would just scale the icon according to the page and leave rest of the implementation as it is.
![image](https://github.com/processing/p5.js-web-editor/assets/93793691/c4bd4c41-8019-4bb4-8dbd-fbb2783bca87)
